spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/SpringBootConfigurationFinder.java

```diff
@@ -33,20 +34,27 @@
  *
  * @author Phillip Webb
  */
-final class SpringBootConfigurationFinder {
+public final class SpringBootConfigurationFinder {
 
 	private static final Map<String, Class<?>> cache = Collections
 			.synchronizedMap(new Cache(40));
 
 	private final ClassPathScanningCandidateComponentProvider scanner;
 
-	SpringBootConfigurationFinder() {
+	public SpringBootConfigurationFinder() {
 		this.scanner = new ClassPathScanningCandidateComponentProvider(false);
```

Comment:
Rather than a copy/paste, this should call the other constructor you've created with `SpringBootConfiguration.class`